src/SprykerEco/Zed/Payone/Persistence/PayonePersistenceFactory.php

```diff
@@ -79,4 +81,12 @@ public function createPayonePersistenceMapper(): PayonePersistenceMapper
     {
         return new PayonePersistenceMapper();
     }
+
+    /**
+     * @return \SprykerEco\Zed\Payone\Dependency\Service\PayoneToUtilEncodingServiceInterface
+     */
+    public function getServiceUtilEncoding(): PayoneToUtilEncodingServiceInterface
```

Comment:
`Service` word should be in the beginning only for the constant.
```suggestion
    public function getUtilEncodingService(): PayoneToUtilEncodingServiceInterface
```